docker/nginx.conf

```diff
@@ -1,12 +1,34 @@
+map $sent_http_content_type $expires {
+    # cache everything for 1 year
+    default 1y;
+    # html files shouldn't be cached for single-page applications
+    text/html off;
+}
+
 server {
     listen 80;
     listen [::]:80;
     server_name localhost;
 
+    # serve compressed file if filename.gz exists
+    gzip_static on;
+
+    # Deny IE browsers from going into quirks mode
+    add_header X-UA-Compatible "IE=Edge";
+    # X-Frame-Options is to prevent from clickJacking attack
+    add_header X-Frame-Options SAMEORIGIN;
+    # This header enables the Cross-site scripting (XSS) filter
+    add_header X-XSS-Protection "1; mode=block;";
+    # disable content-type sniffing on some browsers.
+    add_header X-Content-Type-Options nosniff;
+
     location /console {
         root /usr/share/nginx/html;
         index index.html index.htm;
         try_files $uri /console/index.html;
+        expires $expires;
```

Comment:
I don't seem to see this in the response header

Reply:
maybe the Cache-Control one is causing a conflict?

Reply:
Only will be applied to non `text/html` requests 👍🏻